src/components/map/MapboxMap.js

```diff
@@ -71,6 +72,14 @@ const MapboxMap = () => {
     redrawPolygons(polygons);
   }, [currentMapId, unsavedMapUuid]);
 
+  // Load map from session storage if it exists
+  useEffect(() => {
```

Comment:
There's a `useEffect` block in MapApp.js that does a lot of initial actions, such as getting the list of maps and establishing a websocket connection. It's the highest level component so probably the most suitable place to do stuff like this. So I think it would make sense to move this logic there.

And we could also remove the value from session storage when the user becomes unauthenticated within this same block, to avoid some weird behaviour. For example, if the user logs out whilst viewing a map, then logs back in to a different account, the code as it is would try to take them to the same map.

Reply:
We could also achieve the same behaviour with existing actions, to avoid duplicating code in a new `loadMapFromStorageId` method. I think this is preferable so that if in the future we want to change the logic in `openMap`, we only have to do it one place.